gallery/src/index.tsx

```diff
@@ -8,14 +8,11 @@ import reportWebVitals from './reportWebVitals';
 // the app so all of our components can inherit the styles
 import '@neo4j-ndl/base/lib/neo4j-ds-styles.css';
 
-
-const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
-);
+const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
 root.render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
+  </React.StrictMode>,
```

Comment:
As the TSX block was an argument in the render function the Prettier was trying to add a coma at the end, I updated the Prettier rules to avoid this behaviour :)